bolt-sidecar/src/state/execution.rs

```diff
@@ -100,6 +114,27 @@ pub struct ExecutionState<C> {
     kzg_settings: EnvKzgSettings,
     /// The state fetcher client.
     client: C,
+    /// Constant values used for validation
+    constants: Constants,
+}
+
+/// Constants used for validation.
+#[derive(Debug)]
+pub struct Constants {
+    block_gas_limit: u64,
+    max_tx_input_bytes: usize,
+    #[allow(unused)]
+    max_init_code_byte_size: usize,
+}
+
+impl Default for Constants {
+    fn default() -> Self {
+        Self {
+            block_gas_limit: 30_000_000,
+            max_tx_input_bytes: 4 * 32 * 1024, // 128kb
+            max_init_code_byte_size: 2 * 24576,
+        }
+    }
 }
```

Comment:
Any strong reason you're putting these constants in a struct field compared to being just constants in the code?

Reply:
Because these should be configurable in theory, but I'm not fully convinced if this is useful now.
What makes more sense in your opinion?

Reply:
Why should they be configurable in theory? Just curious, I thought this values could change only with an hard-fork

Reply:
these values are chosen by the EL, and can be configured via CLI, without hardforks